packages/core/src/client.ts

```diff
@@ -302,8 +303,20 @@ export abstract class Client {
     return this
   }
 
+  /**
+   * @deprecated Use {@link event} instead.
+   */
   logEvent(data: Record<string, unknown>): void {
-    this.__eventsLogger.logEvent(data)
+    logDeprecatedMethod(this.logger, 'Honeybadger.logEvent', 'Honeybadger.event')
```

Comment:
`logEvent` will be shown as deprecated on IDEs and will also write a warn message at every 100 method calls.